app/bundles/ApiBundle/Controller/CommonApiController.php

```diff
@@ -517,6 +517,18 @@ public function initialize(FilterControllerEvent $event)
     }
 
     /**
+     * Creates new entity from provided params.
+     *
+     * @param array $params
+     *
+     * @return object
+     */
+    public function getNewEntity(array $params)
```

Comment:
The parameter is not used in the method?

Reply:
Not in the `CommonApiController`, but I need to get the `Tag` entity based on the tag name to ensure there are no duplicates and for that I need to know the value from `$params` in the `TagApiController`.

The other option was to overwrite the `newEntityAction` and `newEntitiesAction`, but it would lead to code duplication.

Reply:
Uf, this is a great example of inheritance mess. But ok, so this is not related to this PR.
Approved :)